install/installer/pkg/components/openfga/deployment.go

```diff
@@ -137,8 +72,108 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 				}},
 			},
 		}...)
+
+		// We use our cloud-sql-proxy sidecar to target the DB.
+		dbHost := "localhost"
+		openfgaEnvVars = append(openfgaEnvVars, []corev1.EnvVar{
+			{
+				Name:  "OPENFGA_DATASTORE_ENGINE",
+				Value: "mysql",
+			},
+			{
+				Name: "DB_PASSWORD",
+				ValueFrom: &corev1.EnvVarSource{SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: cfg.CloudSQL.DatabaseSecretRef,
+					},
+					Key: "password",
```

Comment:
How will the real username and password be provided to the env var?

Reply:
Through a k8s secret, which has been added in this PR https://github.com/gitpod-io/ops/pull/7712

TF loads the secret from GCP into k8s, we reference it through the `cfg.CloudSQL.DatabaseSecretRef` config